CHANGES.rst

```diff
@@ -1,6 +1,19 @@
 ﻿Changelog
 =========
 
+Changes in Apache Libcloud in development
+-----------------------------------------
+
+Compute
+~~~~~~~
+
+- [OpenStack] Fix error attaching a Floating IP to an OpenStack node does not
```

Comment:
I this there is a word missing in this sentence / similar since it doesn't seem to make sense.